packages/pods-core/src/builtin/NextjsExportPod.ts

```diff
@@ -493,6 +516,25 @@ export class NextjsExportPod extends ExportPod<NextjsExportConfig> {
       config: engineConfig,
       noExpandSingleDomain: true,
     });
+
+    const sidebarPath =
+      "sidebarPath" in siteConfig ? siteConfig.sidebarPath : undefined;
+
+    const sidebarsResult = getSidebars(
```

Comment:
isn't this still just one sidebar? why the plural?

Reply:
In this case it would return actually the map of multiple sidebars. inside `TreeUtils.generateTreeData` I had it trimmed to only the first one so it would actually only support a single sidebar.